src/visitor/visitor_util.rs

```diff
@@ -69,7 +69,7 @@ pub fn get_dd_paren_expr(
         call
     } else {
         assignations.push(call);
-        return Expr::Paren(ParenExpr {
+        Expr::Paren(ParenExpr {
```

Comment:
Linter failed here, some new linter version I guess. That's why I am changing this.